### DISCUSSION TITLE
Stream Priority

### DISCUSSION THREAD

**Discussion by samuelryberg** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#177] [Q&A]
If i put Priority.POINTS_DESCEDING and twitch_miner.mine([Streamer("streamer1"), Streamer("streamer2")], followers=True)

This will make my bot watch streamer1 and streamer2 if they are online and if they are not online it will go to the channel where i have the most points (lets call this streamer3) right?

but what if streamer1 and streamer2 was offline when i started the bot but goes online after a while, will the bot switch from streamer3 that was being mined before streamer1/2 went online or will it stay on streamer3 until they go offline and then go to streamer 1/2?

**Comment by Rakambda** [comment]
It should switch to the users with the most points (so streamer3 and another one will be mined).

The priority rules are applied regularly against the list of streamers online.

So if you have:
Streamer1 - 100pts
Streamer2 - 200pts
Streamer3 - 300pts

You start the bot and streamer 1/2 are online, they'll be mined.
When Streamer3 comes online, the bot will mine Streamer 3/2

**Reply by samuelryberg** [reply]
alright sweet :)

also what would happen if i open a stream in my browser while the bot is running? will that mess it up or what would happen?

**Reply by Rakambda** [reply]
The bot will continue it's thing like if you'd have several tabs opened in your browser I suppose.

However twitch is limiting the number of channels you get points for to 2 concurrently. So in the end it's twitch that will decide which will receive points (between the 2 mined by the bot and the ones you have opened in the browser).
